### PR TITLE
New home page

### PR DESCRIPTION
**Pull request by tetogomez**
### New home page

Refactor home page (Add new styling)

### Steps to test

1. run `make run`
2. go to `localhost:3000`
3. test home page

### Checklist
- [ ] I have added/updated tests to cover these changes.
- [ ] The branch name, commit history and PR title follow [conventions](https://developers.eoscostarica.io/docs/open-source-guidelines#pull-request-general-guidelines).
- [ ] I have taken into consideration any impact to site performance.

### CODE REVIEW

File: webapp/src/language/en.json [hunk 1]
```diff
@@ -204,7 +171,17 @@
...
+    "legend2": "This dApp is open-source",
+    "home": "Home",
+    "blockProducers": "Block Producers",
+    "networkMonitor": "EOS Network Monitor",
+    "proxies": "Proxies",
+    "about": "About EOS Rate",
+    "ricardianContract": "Ricardian Contract",
+    "help": "Help",
+    "madeBy": "Made for the community by EOS Costa Rica",
+    "eosRate": "EOSRate.io 2021"
```
Comment (xavier506) [inline webapp/src/language/en.json:184]: can we set the year as a parameter? it will soon be 2022 :christmas_tree:

**Review by xavier506 — APPROVED** [review]

File: webapp/src/routes/Home/index.js [hunk 2]
```diff
@@ -1,45 +1,152 @@
...
+        <Typography className={classes.videoTitle}>{t('tour')}</Typography>
+        <Box className={classes.video}>
+          <Video src={t('videoUrl')} />
+        </Box>
+        <Link
+          component='button'
+          variant='body2'
+          className={classes.link}
+          onClick={() => {
+            console.info("I'm a button.")
```
Comment (AngeloCG97) [inline webapp/src/routes/Home/index.js:75]: This console.info is necessary?

File: webapp/src/routes/Home/index.js [hunk 3]
```diff
@@ -1,45 +1,152 @@
...
+        </Link>
+      </Box>
+      <Box className={classes.blockProducers}>
+        <Typography className={classes.votes}>{t('mostVoted')}</Typography>
+        <Link
+          component='button'
+          variant='body2'
+          className={classes.bpList}
+          onClick={() => {
+            console.info("I'm a button.")
```
Comment (AngeloCG97) [inline webapp/src/routes/Home/index.js:88]: This console.info is necessary?

File: webapp/src/routes/Home/index.js [hunk 4]
```diff
@@ -1,45 +1,152 @@
...
-      >
-        <RateCategory />
+      <Box className={clsx(classes.blockProducers, classes.twitter)}>
+        <Typography className={classes.votes}>{t('mention')}</Typography>
+        <Link
+          component='button'
+          variant='body2'
+          className={classes.bpList}
+          onClick={() => {
+            console.info("I'm a button.")
```
Comment (AngeloCG97) [inline webapp/src/routes/Home/index.js:132]: This console.info is necessary?

File: webapp/src/components/Header/index.js [hunk 5]
```diff
@@ -149,6 +152,44 @@ const Header = memo(({ onDrawerToggle }) => {
...
+      <AppBar className={classes.appBarHome} position='sticky'>
+        <Toolbar className={classes.toolbar}>
+          <Box className={classes.boxLogoHome}>
+            <IconButton aria-label='Open drawer' onClick={onDrawerToggle}>
+              <MenuIcon />
+            </IconButton>
+            <Typography>Menu</Typography>
+          </Box>
+          <Box className={clsx(classes.homeSvgs, classes.desktopSection)}>
+            <Box style={{ marginRight: '10px', marginTop: '5px' }}>
```
Comment (AngeloCG97) [inline webapp/src/components/Header/index.js:166]: We can remove this inline style

File: webapp/src/components/Header/index.js [hunk 6]
```diff
@@ -149,6 +152,44 @@ const Header = memo(({ onDrawerToggle }) => {
...
+            <LanguageSelector />
+            <AuthButton
+              loading={state.loadingLogin}
+              user={state.user}
+              onLogin={handleLogin}
+              onSignOut={handleSignOut}
+            />
+          </Box>
+          <Box className={clsx(classes.homeSvgs, classes.mobileSection)}>
+            <Box style={{ marginTop: 8 }}>
```
Comment (AngeloCG97) [inline webapp/src/components/Header/index.js:178]: We can remove this inline style

Comment (AngeloCG97) [reply webapp/src/components/Header/index.js:178]: Can use <Box mt={1}>

**Pull request merged**